rotkehlchen/externalapis/cryptocompare.py

```diff
@@ -480,6 +480,28 @@ def query_endpoint_price(
         result = self._api_query(path=query_path)
         return result
 
+    def query_current_price(
```

Comment:
Because `query_endpoint_price` returns `Dict[str,Any]`, I want to return `Price` and `self._special_case_handling` for `query_endpoint_price` returns a dict. I didn't want to mess up with the latter.
If I'm allowed to return a Price here (like the `query_endpoint_pricehistorical` case) I will change the function name and remove the wrapper.

Reply:
Sure you are. Just check where what you are working on is used. This is only used here. There is no need to keep the Dict[str, Any] return. Why would you need to do that?